packages/gafl-webapp-service/src/server.js

```diff
@@ -63,8 +70,8 @@ const plugIns = [
        * 31 of the GDS template. This will probably come up as an advisory in the PEN test.
```

Comment:
Probably don't need this comment about PEN test anymore.

Reply:
removed